app/src/main/java/id/rizmaulana/covid19/ui/adapter/DetailAdapter.kt

```diff
@@ -75,4 +92,9 @@ class DetailAdapter(
         notifyDataSetChanged()
     }
 
+    interface ActionListener {
```

Comment:
Mas @fajarnuha , this interface function already replace by higher order function, isn't it? can we just remove this?

Reply:
sure, i forget to remove this one